schulze_condorcet/schulze_condorcet.py

```diff
@@ -139,13 +174,77 @@ def _subindex(alist: Collection[Container[str]], element: str) -> int:
         winners = _schulze_winners(d, remaining)
         result.append(winners)
 
-    # Return the aggregated preference list in the same format as the input votes are.
-    condensed = Vote(">".join("=".join(level) for level in result))
-    detailed = []
+    return counts, d, result
```

Comment:
Yes, this is true. However, maybe one does in the future? But yeah, since this is an internal helper and no interface function, we can change it if we ever need d...